techmatters/flex-plugins
# Adding spanish translation files

Primary reviewer: @GPaoloni 

## Description
This PR adds 2 new language files for spanish and it modifies pluginHelpers.js to include spanish as a language.


### Verification steps

Set "counselorLanguage":"es-ES" in your worker attributes and refresh the Agent Desktop.
The flex UI should show the "general" forms in spanish, excluding forms managed by Flex.

![image](https://user-images.githubusercontent.com/11992408/171727059-52e7f6db-c864-4db3-9100-118f95c81584.png)
